bsp/nuvoton/libraries/m480/StdDriver/SConscript

```diff
@@ -0,0 +1,26 @@
+# RT-Thread building script for component
+Import('rtconfig')
+from building import *
+
+cwd = GetCurrentDir()
+src = []
+libs = ['']
+libpath = [cwd + '/lib']
+cpppath = [cwd + '/inc']
+
+if GetDepend('BSP_USE_STDDRIVER_SOURCE'):
+	src = Glob('*src/*.c') + Glob('src/*.cpp')
+	group = DefineGroup('m480_stddriver', src, depend = [''], CPPPATH = cpppath)
+else:
+	if rtconfig.CROSS_TOOL == 'keil':
```

Comment:
We provide the M480 StdDriver library and source at the same time, because we wish RTT user decrease building time for increase user-experience.
[Default ](https://github.com/OpenNuvoton/rt-thread/blob/m480/bsp/nuvoton/libraries/m480/rtt_port/Kconfig#L11)value is library, If user want to rebuild these sources, they can select option.